WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -564,6 +622,13 @@ import WordPressComAnalytics
     }
 
 
+    /// Displays the options menu for the specifed post.  On the iPad the menu
+    /// is displayed as a popover from the anchorview.
+    ///
+    /// - Parameters:
+    ///     - post: The post in question.
+    ///     - anchorView: The view to anchor a popover.
```

Comment:
The external parameter name here is `fromView`, not `anchorView`. Same for the following method too.